### PR TITLE
feat(pie-icons-configs): DSW-899 remove dependency on css size classes for regular icon sizes

### PR DESCRIPTION
**Pull request by dandel10n**
Aligns the logic of size application for regular and large icons.
Now `iconSize` prop value for regular icons is mapped to size values.
Removes the need to write css styles in consuming apps for icon sizes.
Moves `getSvgProps` logic to configs.js from react and vue configs.
`pie-icons-vue` and `pie-icons-react` rebuilt to apply the changes.

Tested by yalcing react icon package to snacks and vue icon package to fozzie-components

### CODE REVIEW

File: packages/tools/pie-icons-configs/configs.js [hunk 1]
```diff
@@ -56,17 +56,48 @@ export function validateGetLargeIconSize (iconSizeValue) {
...
+    const iconSize = sizeToValueMap[iconSizeValue];
 
-    return { isValid: true, iconSizeClass };
+    return { isValid: true, iconSize };
 }
+
+/**
+ * Returns props for the svg element based on the provided parameters
+ * @param {string} svgClasses - String of classes assigned to the icon
+ * @param {string} staticClass - String of classes assigned to the component
```
Comment (ashleynolan) [inline packages/tools/pie-icons-configs/configs.js:77]: Small one – should this be `staticClasses`? Can't work out if this could potentially be multiple or not 😆

Comment (dandel10n) [reply packages/tools/pie-icons-configs/configs.js:77]: Yes, it can be a string of multiple class names 👍

**Review by ashleynolan — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

File: packages/tools/pie-icons-configs/README.md [hunk 2]
```diff
@@ -40,4 +40,4 @@ You can run the tests with `yarn test`.
 
 ## Contributing
 
-Before starting please read our [contributing guide](https://vue.pie.design/?path=/story/documentation-getting-started-contributing--page)
+Before starting please read our [contributing guide](https://pie.design/engineers/contributing/)
```
Comment (xander-marjoram) [inline packages/tools/pie-icons-configs/README.md:43]: While you're here can we update the test script above to `yarn test --filter=pie-icons-configs`?

File: packages/tools/pie-icons-configs/configs.js [hunk 3]
```diff
@@ -56,17 +56,48 @@ export function validateGetLargeIconSize (iconSizeValue) {
...
+ * @param {string} staticClasses - String of classes assigned to the component
+ * @param {{string|number}} iconSizeValue - Value of the iconSize prop
+ * * @param {string} componentName - Name of the component
+ * @returns {Object} - Object of props to be assigned to the svg element
+ */
+
+export const getSvgProps = (svgClasses, staticClasses, iconSizeValue, componentName) => {
+    const isLargeIcon = svgClasses.endsWith('Large') || svgClasses.endsWith('-large');
+
+    const { isValid, iconSize: validIconSize } = isLargeIcon
```
Comment (xander-marjoram) [inline packages/tools/pie-icons-configs/configs.js:86]: Nit: Is this rename necessary? Or if you prefer this name then maybe you could change it in the validation methods

File: packages/tools/pie-icons-vue/README.md [hunk 4]
```diff
@@ -70,6 +70,9 @@ Example:
 <icon-alert-triangle-large iconSize="40" />
 ```
 
+### Styles
+Vue icons package doesn't provide css styles. Css classes `c-pieIcon c-pieIcon--{iconName}` are available for consuming applications to add styles needed for the their project.
```
Comment (xander-marjoram) [inline packages/tools/pie-icons-vue/README.md:74]:
```suggestion
The Vue icons package doesn't provide CSS styles. CSS classes `c-pieIcon c-pieIcon--{iconName}` are available for consuming applications to add the styles they require.
```

File: packages/tools/pie-icons-vue/test/IconAlcohol.test.js [hunk 5]
```diff
@@ -49,22 +52,25 @@ describe('IconAlcohol (Regular variant) ::', () => {
...
         const wrapper = mount(IconAlcohol, { propsData });
 
         // Assert
-        expect(wrapper.classes()).not.toContain(`c-pieIcon--${iconSize}`);
-        expect(wrapper.classes()).toContain('c-pieIcon--xs'); // Default size
-        expect(errorMock).toHaveBeenCalledTimes(1);
+        expect(wrapper.html()).toContain(`width="${defaultIconSize}"`);
+        expect(wrapper.html()).toContain(`height="${defaultIconSize}"`);
+        expect(spy).toHaveBeenCalled();
+        spy.mockRestore();
```
Comment (xander-marjoram) [inline packages/tools/pie-icons-vue/test/IconAlcohol.test.js:67]: Was this necessary even with the `afterEach`?

File: packages/tools/pie-icons-vue/test/IconAlcoholLarge.test.js [hunk 6]
```diff
@@ -90,7 +89,8 @@ describe('IconAlcoholLarge (Large variant) ::', () => {
             // Assert
             expect(wrapper.html()).toContain('width="32"');
             expect(wrapper.html()).toContain('height="32"');
-            expect(errorMock).toHaveBeenCalledTimes(1);
+            expect(spy).toHaveBeenCalled();
+            spy.mockRestore();
```
Comment (xander-marjoram) [inline packages/tools/pie-icons-vue/test/IconAlcoholLarge.test.js:93]: This approach is also good. If we go down this path we can probably remove the `afterEach` blocks.

**Review by xander-marjoram — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]
